nep-x1.mediawiki

```diff
@@ -0,0 +1,161 @@
+<pre>
+  NEP: Undefined
+  Title: NFT Royalty Standard
+  Author: Owen Zhang<zhangzhihong@ngd.neo.org>, Vincent Geneste<vincentgeneste@gmail.com>
+  Type: Standard
+  Status: Draft
+  Created: 2022-09-05
+</pre>
+
+==Abstract==
+
+This NEP defines a global standard to get royalty payment information for non-fungible tokens (NFTs) to enable support for royalty payments across all NFT marketplaces in the NEO Smart Economy.
```

Comment:
This NEP defines a global standard to get royalty payment information for Non-Fungible Tokens (NFTs) in order to enable support for royalty payments across all NFT marketplaces in the NEO Smart Economy.

Reply:
minor changes here as well, fell free to apply or not